homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -16,9 +16,24 @@
 from homeassistant.components.bluetooth import BluetoothServiceInfo
 from homeassistant.config_entries import SOURCE_BLUETOOTH, ConfigFlow, ConfigFlowResult
 from homeassistant.const import CONF_ADDRESS, CONF_CLIENT_ID, CONF_PIN
+from homeassistant.helpers.device_registry import format_mac
 
 from .const import DOMAIN, LOGGER
 
+BLUETOOTH_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_PIN): str,
+    }
+)
+
+USER_SCHEMA = BLUETOOTH_SCHEMA.extend(
+    {
+        vol.Required(CONF_ADDRESS): str,
+    }
+)
+
+REAUTH_SCHEMA = BLUETOOTH_SCHEMA
+
```

Comment:
This is a nice cleanup 👍
It's not aligned with the PR title, so the same comment as on the retry behavior applies.